megatron/initialize.py

```diff
@@ -349,9 +349,12 @@ def _warmup_jit_function():
         dtype = torch.float32
 
     # Warmup fused bias+gelu
+    seq_length = args.seq_length
+    if args.ds_sequence_parallel_fpdt:
+        seq_length = 8192
```

Comment:
Can you define this as another variable like "FPDT_SEQ_LEN" and give a description in a comment why we have this setting?

Reply:
This is fixed by setting it to be `ds_sequence_parallel_fpdt_chunk_size` if FPDT is enabled.